unsupported/Eigen/CXX11/src/Tensor/TensorReductionGpu.h

```diff
@@ -7,23 +7,27 @@
 // Public License v. 2.0. If a copy of the MPL was not distributed
 // with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-#ifndef EIGEN_CXX11_TENSOR_TENSOR_REDUCTION_CUDA_H
-#define EIGEN_CXX11_TENSOR_TENSOR_REDUCTION_CUDA_H
+#ifndef EIGEN_CXX11_TENSOR_TENSOR_REDUCTION_GPU_H
+#define EIGEN_CXX11_TENSOR_TENSOR_REDUCTION_GPU_H
+
+#if defined(EIGEN_HIPCC)
+#define warpSize (64)
```

Comment:
see comment in `TensorConvolutionGpu.h`. should `warpSize` be aggregated in one place?